### PR TITLE
Service: Analog Measurement

### PR DESCRIPTION
**Pull request by jamesadevine**
This pull request defines a new service.

### CODE REVIEW

File: services/analogmeasurement.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+        Differential = 1
+    }
+
+    const measurement_type: ADCReportType @ 0x181
+The type of measurement that is taking place. Absolute results are measured with respect to ground, whereas differential results are measured against another signal that is not ground.
+
+    const measurement_name: string @ 0x182
+A string containing the net name that is being measured e.g. `POWER_DUT` or a reference e.g. `DIFF_DEV1_DEV2`. These constants can be used to identify a measurement in program code.
+
+    ro volatile measurement: f32 @ 0x183
```
Comment (pelikhan) [inline services/analogmeasurement.md:21]: reading?

File: services/analogmeasurement.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+        Differential = 1
+    }
+
+    const measurement_type: ADCMeasurementType @ 0x181
+The type of measurement that is taking place. Absolute results are measured with respect to ground, whereas differential results are measured against another signal that is not ground.
+
+    const measurement_name: string @ 0x182
+A string containing the net name that is being measured e.g. `POWER_DUT` or a reference e.g. `DIFF_DEV1_DEV2`. These constants can be used to identify a measurement in program code.
+
+    ro volatile measurement: f32 @ reading
```
Comment (pelikhan) [inline services/analogmeasurement.md:21]: volatile is implicit for reading

Comment (jamesadevine) [reply services/analogmeasurement.md:21]: ok, so you want me to remove volatile?

File: services/analogmeasurement.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+        Differential = 1
+    }
+
+    const measurement_type: ADCMeasurementType @ 0x181
+The type of measurement that is taking place. Absolute results are measured with respect to ground, whereas differential results are measured against another signal that is not ground.
+
+    const measurement_name: string @ 0x182
+A string containing the net name that is being measured e.g. `POWER_DUT` or a reference e.g. `DIFF_DEV1_DEV2`. These constants can be used to identify a measurement in program code.
+
+    ro measurement: f32 @ reading
```
Comment (mmoskal) [inline services/analogmeasurement.md:22]: I guess no harm upgrading to `f64`?  also you want unit, `V`

**Pull request merged**